scripts/bootloader/hash.py

```diff
@@ -8,22 +8,26 @@
 import hashlib
 import sys
 import argparse
+from intelhex import IntelHex
 
 
 def parse_args():
     parser = argparse.ArgumentParser(
         description="Hash data from stdin or file.",
```

Comment:
Change description and help text to reflect that stdin is no longer supported. (or add stdin support back :))

Reply:
Will change description as there is no mechanism two separate between hex and bin as being given to stdin. And if the user has to give that separation, they just as might send the path to the file.